src/pages/availability-panel/index.tsx

```diff
@@ -36,30 +41,11 @@ const AvailabilityPanel: FC = () => {
                 setIsTaskLoading(false);
             }
         };
-        const fetchIdleUsers = async () => {
-            try {
-                const url = `${process.env.NEXT_PUBLIC_BASE_URL}/members/idle`;
-                const { requestPromise } = fetch({ url });
-                const fetchPromise = await requestPromise;
-                const { idleMemberUserNames } = fetchPromise.data;
-                const filterMembers = idleMemberUserNames.filter(
-                    (username: string) => username
-                );
-                const sortedIdleMembers = filterMembers.sort();
-                setIdleMembersList(sortedIdleMembers);
-                setError(false);
-            } catch (Error) {
-                setError(true);
-            } finally {
-                setIsMemberLoading(false);
-            }
-        };
         fetchTasks();
-        fetchIdleUsers();
     }, [refreshData]);
 
     let isErrorOrIsLoading;
-    if (error) {
+    if (error || membersError) {
         isErrorOrIsLoading = (
             <span className={classNames.statusMessage}>
                 Something went wrong, please contact admin!
```

Comment:
if we have membersError then why are we using error

Reply:
As there are two different fetch calls happening, if we use same variable for both the errors I feel it will increase the code complexity unnecessarily.

Reply:
please see the solution in terms of rtk, we are removing fetch calls, in the next 2 or 3 PR's fetch calls will be removed so think in terms of rtk

Reply:
Yes since each RTK query returns a separate error variable the logic will remain same in the future.